Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -69,76 +73,98 @@
 
 - type: alert
   id: Weightless
-  icon: /Textures/Interface/Alerts/Weightless/weightless.png
+  icons: [ /Textures/Interface/Alerts/Weightless/weightless.png ]
   name: Weightless
   description: >
     Gravity has ceased affecting you, and you're floating around aimlessly. Find something sturdy to hold onto, or throw or shoot something in a direction opposite of you.
     Mag-boots or jetpacks would help you move with more control
 
 - type: alert
   id: Stun
-  icon: /Textures/Objects/Weapons/Melee/stunbaton.rsi/stunbaton_off.png #Should probably draw a proper icon
+  icons: [ /Textures/Objects/Weapons/Melee/stunbaton.rsi/stunbaton_off.png ] #Should probably draw a proper icon
   name: "[color=yellow]Stunned[/color]"
   description: "You're [color=yellow]stunned[/color]! Something is impairing your ability to move or interact with objects"
 
 - type: alert
   id: Handcuffed
   onClick: !type:RemoveCuffs  { }
-  icon: /Textures/Interface/Alerts/Handcuffed/Handcuffed.png
+  icons: [ /Textures/Interface/Alerts/Handcuffed/Handcuffed.png ]
   name: "[color=yellow]Handcuffed[/color]"
   description: "You're [color=yellow]handcuffed[/color] and can't use your hands. If anyone drags you, you won't be able to resist."
 
 - type: alert
   id: Buckled
   category: Buckled
   onClick: !type:Unbuckle  { }
-  icon: /Textures/Interface/Alerts/Buckle/buckled.png
+  icons: [ /Textures/Interface/Alerts/Buckle/buckled.png ]
   name: "[color=yellow]Buckled[/color]"
   description: "You've been [color=yellow]buckled[/color] to something. Click the alert to unbuckle unless you're [color=yellow]handcuffed.[/color]"
 
 - type: alert
   id: HumanCrit
   category: Health
-  icon:
-    sprite: /Textures/Interface/Alerts/human_health.rsi
+  icons:
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
     state: health6
   name: "[color=red]Critical Condition[/color]"
   description: "You're severely injured and unconscious."
 
 - type: alert
   id: HumanDead
   category: Health
-  icon:
-    sprite: /Textures/Interface/Alerts/human_health.rsi
+  icons:
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
     state: health7
   name: Dead
   description: You're dead, note that you can still be revived!
 
 - type: alert
   id: HumanHealth
   category: Health
-  icon:
-    sprite: /Textures/Interface/Alerts/human_health.rsi
-    state: health
+  icons:
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health0
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health1
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health2
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health3
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health4
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health5
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health6
+  - sprite: /Textures/Interface/Alerts/human_health.rsi
+    state: health7
```

Comment:
Each severity now has an explicit icon. On one hand, this clearly pretty shit for alerts with many severities. But on the other hand, this allows icons to be sourced from different RSIs/pngs and allows them to have nonconsecutive names. + yaml validation.

Reply:
this is worth it imo